src/record-matching/bib/generate-query-list.js

```diff
@@ -33,12 +33,15 @@ export function generateIdentifierQueries(record) {
 	function getStandardIdentifiers() {
 		return record.get(/^020|022|024$/)
 			.reduce((acc, field) => {
-				const subfield = field.subfields.find(sf => sf.code === 'a');
+				const codes = ['a', 'z'];
+				field.subfields
+					.filter(sub => {
```

Comment:
The purpose `filter` is to return a more compact list of the original. Use `forEach` to make the code easier to understand.